umea/meetup.yaml

```diff
@@ -0,0 +1,82 @@
+city: Umeå
+country: sweden
+meetupID: Cloud-Native-Northern-Sweden
+meetups:
+- address: Tvistevägen 47
+  attendees: 19
+  date: "2018-05-28T17:00:00Z"
+  duration: 2h0m0s
+  id: 261450080
+  name: Tremds in cloud native - a summary from KubeCon + CloudNativeCon Barcelona
+  presentations:
+  - endTime: "18:30"
+    slides: null
+    speakers: 
+      - Linda Almersson
```

Comment:
for every speaker and company mentioned here, can you append these to `companies.yaml` and `speakers.yaml` top-level in the repo?

See e.g. #56 of how to do this. Sorry, we're lacking documentation of this I know

Reply:
Done in latest commit, I'll rebase